pkg/tsdb/azuremonitor/applicationinsights-datasource_test.go

```diff
@@ -290,3 +210,21 @@ func TestAppInsightsPluginRoutes(t *testing.T) {
 	}
 
 }
+
+func TestInsightsDimensionsUnmarshalJSON(t *testing.T) {
+	var err error
+	a := []byte(`"foo"`)
+	b := []byte(`["foo"]`)
+	_ = b
```

Comment:
What's the point of this? To stop the compiler from complaining about `b` being unused?

Reply:
Oopps, yes, will remove. Was while I was messing with the func (same reason I made a var err error)`.